src/components/Header/index.js

```diff
@@ -11,6 +11,10 @@ class Header extends Component {
     openMenu: true
   };
 
+  redirectToReferral(){
+    window.location.pathname = "/referral";
```

Comment:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/history.md 

Use `this.props.history.push('/referral')`. I would use a name like onClickReferral or handleReferral/(Redirect)

Reply:
This is probably change a lot of code, so I'll leave it like this and make a branch to implement it separately. I will change the function name, though.